server/routes/offenceCodeDecisions/decisionForm.ts

```diff
@@ -1,20 +1,24 @@
-// Form data backing a decision. Used in form submits and to allow the saving of form data on a decision page so that
-// we can repopulate it when returning from a user lookup.
+// Form data backing a decision. Used in form submits and to allow the saving of form data on a decision page. This
+// allows consistent validation and saving backing state when navigating away from a page to do e.f. prisoner search.
 export type DecisionForm = {
   selectedDecisionId?: string
-  selectedDecisionData?: StaffData | OfficerData | PrisonerData
+  selectedDecisionData?: StaffData | OfficerData | PrisonerData | AnotherData
 }
-export type SelectPrisonerData = {
-  userId?: string
-  userSearchInput?: string
+export type StaffData = {
```

Comment:
Perhaps these should go in the "Helper" classes along with the data to process them?